### ISSUE TITLE
Copy memoized fn's doc when using `memoize`

### ISSUE THREAD

**Issue by simonacca** [clojure-lsp/clojure-lsp#1863]
Clojure-lsp has a ["documentation and clojuredocs integration"](https://clojure-lsp.io/features/#documentation-and-clojuredocs-integration) feature which (among other things) can show a function's signature and documentation.

When a function is memoized using `memoize` [from the standard library](https://clojuredocs.org/clojure.core/memoize) and stored in a var, it would be useful if the the function's documentation were to be carried over to memoized version of the fn.

**Comment by ericdallo** [comment]
I agree it'd be good for user but it's not trivial to do that AFAICS
